1.8/administration/access-node/tunnel.md

```diff
@@ -0,0 +1,113 @@
+---
+post_title: Access by Proxy
+menu_order: 10
+---
+
+When developing applications on DC/OS, you may find it helpful to access your cluster via SOCKS proxy, HTTP proxy, or VPN. For instance, you can work from your own development environment and immediately test on your DC/OS cluster.
+
+# SOCKS
+dcos tunnel can run a SOCKS proxy over SSH to the cluster. SOCKS proxies work for any kind of network protocol on any port, but your application must be configured to use the proxy, which runs on port 1080 by default. Access URLs by appending the `mydcos.directory` domain.
+
+# HTTP
+Run the HTTP proxy on your local port 80 to tunnel HTTP requests without modifying your application. Access URLs by appending the `mydcos.directory` domain. You can also [use DNS SRV records as if they were URLs](#srv).
+
+<a name="srv"></a>
+## SRV Records
+An SRV DNS record is a mapping from a name to a IP/port pair. DC/OS creates SRV records in the form `_<port-name>._<task-name>._tcp.marathon.mesos.mydcos.directory`. The HTTP proxy exposes these as URLs. This feature can be useful for communicating with Marathon apps.
+
+# VPN
+dcos tunnel provides you with full access to the DNS, masters, and agents from within the cluster. OpenVPN requires root privileges to configure these routes.
+
+# Using dcos tunnel
+
+## Prerequisites
+* DC/OS 1.8+
+* Docker
+* Requires [SSH access](/1.8/administration/access-node/sshcluster/) (key authentication only).
+* For VPN functionality, you must have [the OpenVPN client](https://openvpn.net/index.php/open-source/downloads.html) installed.
+* The dcos tunnel package must be installed: run `dcos package install tunnel-cli --cli` from the DC/OS CLI.
+
+**Note:** * Only Linux and OS X are supported currently.
+
+##  SOCKS
+Run the following command from the DC/OS CLI:
+
+```
+$ dcos tunnel socks
+```
+
+Configure your application to use the proxy on port 1080: `127.0.0.1:1080`
+
+### Port Forwarding
```

Comment:
I prefer to repeat because users don't always read the whole doc and clicking around seems cumbersome.